src/services/openletters/index.jsx

```
@@ -0,0 +1,332 @@
+  addPicTour,
+  getPicTourComments,
+  getPicTourLikes,
+  getPicTourSubCategoryByCategory,
+  likeUnlikePicTour,
+} from "../../app/features/pictours";
+
+import { copyLink } from "../../utils/copyLink";
+import ImagePreviewer from "../../components/previewers/ImagePreviewer";
+
+export const AddOpenLetter = ({
+  setAddModal,
```

ESLint

[error] 'setAddModal' is missing in props validation

(react/prop-types)